src/Shared/DotNetCoreSdkResolver.cs

```diff
@@ -78,11 +76,15 @@ public static bool TryResolveDotNetCoreSdk(IEnvironmentProvider environmentProvi
             {
                 if (!process.Start())
                 {
+                    developmentEnvironment = new DevelopmentEnvironment("Failed to resolve the .NET SDK.  Verify the 'dotnet' command is available on the PATH.");
```

Comment:
Seem to be overloading the usage of `DevelopmentEnvironment` to just return different error messages?